packages/core/src/errors/error-reporter.ts

```diff
@@ -15,15 +15,22 @@ type ErrorReporterInitOptions = {
 	release: string;
 	environment: string;
 	serverName: string;
+	releaseDate?: Date;
 	/**
 	 * Function to allow filtering out errors before they are sent to Sentry.
 	 * Return true if the error should be filtered out.
 	 */
 	beforeSendFilter?: (event: ErrorEvent, hint: EventHint) => boolean;
 };
 
+const SIX_WEEKS_IN_MS = 6 * 7 * 24 * 60 * 60 * 1000;
```

Comment:
Maybe we could move [`Time`](https://github.com/n8n-io/n8n/blob/cat-647-add-dto-for-generate-credential-name-endpoint/packages/cli/src/constants.ts#L119) from cli to core and use that here:

```suggestion
const SIX_WEEKS_IN_MS = 6 * 7 * Time.days.toMilliseconds
```

Reply:
I want to setup a new package `@n8n/constants` for moving all these constants into a single place. but perhaps in another PR.